x/ecocredit/client/testsuite/tx.go

```diff
@@ -1426,6 +1427,84 @@ func (s *IntegrationTestSuite) TestTxBuyDirect() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestUpdateProjectAdmin() {
+	val := s.network.Validators[0]
+	updateTo := s.network.Validators[1].Address.String()
+	clientCtx := val.ClientCtx
+	_, projectId, _ := s.createClassProjectBatch(clientCtx, val.Address.String())
+	cmd := coreclient.TxUpdateProjectAdminCmd()
+	makeArgs := func(projectId, newAdminAddr, from string) []string {
+		args := make([]string, 0, 6)
+		args = append(args, projectId, newAdminAddr, makeFlagFrom(from))
+		return append(args, s.commonTxFlags()...)
+	}
+	unauthorizedAccount, _, err := val.ClientCtx.Keyring.NewMnemonic("unauthorized", keyring.English, sdk.FullFundraiserPath, keyring.DefaultBIP39Passphrase, hd.Secp256k1)
+	s.Require().NoError(err)
+	unauthAddr := unauthorizedAccount.GetAddress()
+	s.fundAccount(clientCtx, val.Address, unauthAddr, sdk.Coins{sdk.NewInt64Coin(sdk.DefaultBondDenom, 50)})
+
+	testCases := []struct {
+		name          string
+		args          []string
+		errMsg        string
+		errLog        string
```

Comment:
had to add 2 separate err strings here, the msg is caught by the CLI itself, Log is caught via the server impl so the expected error is returned in the log of `sdk.TxResponse`